components/Login.tsx

```diff
@@ -0,0 +1,101 @@
+import { View, Text, TouchableOpacity } from "react-native";
+import FloatingTextInput from "../components/FloatingTextInput";
+import axios from "axios";
+import { baseUrl } from "../constants/config";
+import styles from "../constants/styles";
+import { useNavigation } from "@react-navigation/native";
+import { useState } from "react";
+import { SIZES, COLORS } from "../constants/theme";
+
+const Login = () => {
+  const navigation = useNavigation();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+  const [errorEmail, setErrorEmail] = useState("");
+  const [isEmailFilled, setIsEmailFilled] = useState(false);
+
+  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+
+  const handleLogin = () => {
+    axios
+      .post(`${baseUrl}users/login`, {
+        email: email,
+        password: password,
+      })
+      .then(function (response) {
+        if (response.status == 200) {
```

Comment:
Beim Login sollte der zurückgegebene Token in einem "sicheren" Cookie speichern. Laut Kemal gibt es da so HTTP only cookies, welche der Client nicht einsehen kann, ich glaub das ist nen guter einstiegspunkt.

Reply:
Login betrachten wir bei unserem Refactoring diesbezüglich